symfony/messenger/4.1/config/packages/messenger.yaml

```diff
@@ -0,0 +1,9 @@
+framework:
+    messenger:
+        adapters:
+            # Uncomment the following line to enable an adapter named "amqp"
+            # amqp: "%env(MESSENGER_ADAPTER_DSN)%"
+
+        routing:
+            # Route your messages to the senders.
```

Comment:
I would even remove this last dot.

Reply:
Here we go, removed (and actually replaced "senders" with "adapters" as it makes more sense now we the shorter name I believe).